src/cmd/linuxkit/push_azure.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+// Process the run arguments and execute run
+func pushAzure(args []string) {
+	flags := flag.NewFlagSet("azure", flag.ExitOnError)
+	invoked := filepath.Base(os.Args[0])
+	flags.Usage = func() {
+		fmt.Printf("USAGE: %s run azure [options] [name]\n\n", invoked)
+		fmt.Printf("'name' specifies either the name of an already uploaded\n")
+		fmt.Printf("VHD image or the full path to a image file which will be\n")
+		fmt.Printf("uploaded before it is run.\n\n")
+		fmt.Printf("Options:\n\n")
+		flags.PrintDefaults()
+	}
+
+	resourceGroupName := flags.String("resourceGroupName", "", "Name of resource group to be used for VM")
+	accountName := flags.String("accountName", "linuxkitstorage", "Name of the storage account")
+	imagePath := flags.String("imagePath", "", "Local path of the VHD file to be used as OS image")
```

Comment:
this should just be the last argument on the command line, not a flag - see the other push examples.